src/test/java/auth/LoginTestUser.java

```diff
@@ -0,0 +1,87 @@
+package auth;
```

Comment:
The main reasons I know of to mimic the source code package structure in the test code are:
1. Allows you to access protected and package-private methods in the classes under test.
2. Provides a natural organization to the tests.

Neither of these really apply to the CLI tests because the tests will mostly go through the top-level `Main` class instead of calling other internal classes directly. These two tests that I added in this PR are really testing the test code, which is maybe a little weird. But this authentication workaround will be used in almost every single other test, so I thought it would be good to have a couple of dedicated tests so it's easier to narrow down errors to this testing setup code.